zone/exp.cpp

```diff
@@ -530,6 +530,13 @@ void Client::AddEXP(ExpSource exp_source, uint64 in_add_exp, uint8 conlevel, boo
 		aaexp = ScaleAAXPBasedOnCurrentAATotal(GetAAPoints(), aaexp);
 	}
 
+	// Check for AA XP Cap
+	uint64 aaxp_cap = RuleI(AA, MaxAAExpPerKill);
+
+	if (aaxp_cap > 0 && aaexp > aaxp_cap) {
+		aaexp = aaxp_cap;
+	}
+
```

Comment:
```cpp
if (RuleI(AA, MaxAAEXPPerKill) >= 0 && aaexp > RuleI(AA, MaxAAEXPPerKill)) {
	aaexp = RuleI(AA, MaxAAEXPPerKill);
}
```

Reply:
This will allow you to use the `-1` value without worrying about it being an unsigned integer.